cli/resolver.rs

```diff
@@ -200,6 +231,80 @@ impl Resolver for CliGraphResolver {
       }
     }
 
+    if let Some(resolver) =
+      self.npm_resolver.as_ref().and_then(|r| r.as_byonm())
+    {
+      match &result {
```

Comment:
For the future, this is the point where we need to make the most improvement. Essentially, we need a way to determine if a specifier is indeed for an npm package and then we need to ensure we surface the appropriate error messages when it fails.